optimade/server/entry_collections.py

```diff
@@ -13,8 +14,14 @@
 from .mappers import ResourceMapper
 from .query_params import EntryListingQueryParams, SingleEntryQueryParams
 
+ci_force_mongo = os.environ.get("OPTIMADE_CI_FORCE_MONGO", False)
+try:
+    ci_force_mongo = bool(ci_force_mongo)
```

Comment:
Just some initial testing on what `bool` thinks is `True` and `False` arrives at the immediate (maybe wrong) conclusion that this will always result in `True` if the environment variable is set - in any way. However, _only_ if it's not set, it will be `False`.
Is this what you would expect / is this what you see as well?

Reply:
Also - in which case would the `bool()` except?

Reply:
Yep, this is a mistake, there's no way to set this to `False` at the moment (I guess I was hoping that I'd get an int out of `os.environ`, but that wouldn't make much sense). I think the cleanest thing is just to check if the var is `"0"` or `"1"` and error for anything else, given that this is hopefully not going to be used anywhere else :)

Reply:
You could always type-cast it to `int`and catch an exception for that instead perhaps? But in the end it's perhaps cleaner to simply check against hard-coded values like you're suggesting. Not even a need to error for anything else (we can throw warnings) but otherwise just set it to `False` for anything else, i.e., in mock code:
if equal to "1":
  True
else if equal to "0" (or non-existent):
  False
else:
  throw warning that the variable is set, but to non-recognized value, should be either "1" or "0" for True or False, respectively.

Reply:
If you want to be kind, you can even test for several different variations of `True` and `False` ... but maybe not necessary.

Reply:
We're in danger of bikeshedding an internal environment variable that will probably never be touched again X) I'll choose something sensible (probably bool(int(var))) so that you can at least guess what Python is doing with it easily